components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/OAuthAdminService.java

```diff
@@ -478,10 +480,161 @@ public void updateConsumerApplication(OAuthConsumerAppDTO consumerAppDTO) throws
      * @throws IdentityOAuthAdminException
      */
     public String getOauthApplicationState(String consumerKey) throws IdentityOAuthAdminException {
+
         OAuthAppDAO oAuthAppDAO = new OAuthAppDAO();
         return oAuthAppDAO.getConsumerAppState(consumerKey);
     }
 
+    /**
+     * To insert oidc scopes and claims in the related db tables.
+     *
+     * @param tenantId tenant id
+     * @param scope    an oidc scope
+     * @throws IdentityOAuthAdminException if an error occurs when inserting scopes or claims.
+     */
+    public void addScope(int tenantId, String scope, String[] claims)
```

Comment:
Remove tenantId as a parameter in all the APIs and get the tenantID form the loggedInUser